### PR TITLE
[tests-only] Add test to share resource inside of space

### PR DESCRIPTION
**Pull request by grgprarup**
## Description
Added Scenarios:
- Scenario Outline: send share invitation for resource of project space to user with different roles
- Scenario Outline: send share invitation for resource of project space to group with different roles

## Related Issue

- Part of https://github.com/owncloud/ocis/issues/7947

## How Has This Been Tested?

- Locally
- :robot: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Review by saw-jan — APPROVED** [review]
LGTM

File: tests/acceptance/features/apiSharingNg/shareInvitations.feature [hunk 1]
```diff
@@ -2039,3 +2039,178 @@ Feature: Send a sharing invitations
         }
       }
       """
+
+
+  Scenario Outline: send share invitation for resource of project space to user with different roles
```
Comment (SagarGi) [inline tests/acceptance/features/apiSharingNg/shareInvitations.feature:2044]:
```suggestion
  Scenario Outline: send share invitation for project space resource to user with different roles
```

File: tests/acceptance/features/apiSharingNg/shareInvitations.feature [hunk 2]
```diff
@@ -2039,3 +2039,178 @@ Feature: Send a sharing invitations
...
+    Examples:
+      | permissions-role | path          |
+      | Viewer           | textfile1.txt |
+      | File Editor      | textfile1.txt |
+      | Viewer           | FolderToShare |
+      | Editor           | FolderToShare |
+      | Uploader         | FolderToShare |
+
+
+  Scenario Outline: send share invitation for resource of project space to group with different roles
```
Comment (SagarGi) [inline tests/acceptance/features/apiSharingNg/shareInvitations.feature:2125]:
```suggestion
  Scenario Outline: send share invitation for project space resource to group with different roles
```

**Review by SagarGi — APPROVED** [review]
other LGTM :+1:

**Pull request merged**